<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atelier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="ToolBarWidget" name="toolbarWidget" native="true"/>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="LeftWidget" name="leftWidget" native="true"/>
    </item>
    <item row="1" column="4" rowspan="2">
     <widget class="QWidget" name="rightWidget" native="true"/>
    </item>
    <item row="1" column="3" rowspan="2">
     <widget class="QWidget" name="middleWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Welcome to 
 Atelier</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ConnectSettings" name="connectContainer" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="gcodeContainer" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="maintanceContainer" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolBarWidget</class>
   <extends>QWidget</extends>
   <header>toolbarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeftWidget</class>
   <extends>QWidget</extends>
   <header>leftwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectSettings</class>
   <extends>QWidget</extends>
   <header>connectsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
